### DISCUSSION TITLE
Update grammar, tone and style

### DISCUSSION THREAD

**Discussion by UniquePixels** [TheOdinProject/curriculum#25948] [Ideas]
I use Grammarly in my everyday life, at home, and work. It is excellent and gives me a professional and consistent tone.

Would there be any consensus to go through the various lessons and use it or a similar tool to correct grammar and style? It is something I would enjoy doing as a contribution.

**Comment by UniquePixels** [comment]
@ManonLef any thoughts?

**Comment by ManonLef** [comment]
@fabulousgk thanks for wanting to help out!

I don't feel this is a great way to spend time. Both for you and for the review process it will entail. It would of course be a fine tool to use for lessons that stand out or feel confusing due to grammar for example while you come across them. 

We're soon opening up a lot of style guide issues that need addressing for which we probably need quite a bit of contributing. Those will be higher priority since most of the obvious grammar issues mentioned above will be caught on the daily already. We'd love to see your help there if you want!
